docs/shared-content-source/docs/modules/develop/pages/use-flink-streamlets.adoc

```diff
@@ -161,3 +161,55 @@ lazy val taxiRidePipeline = appModule("taxi-ride-pipeline")
 NOTE: If you set `execution.checkpointing.interval` you need to set all other relevant checkpointing settings, the default checkpointing will not be used if `execution.checkpointing.interval` is set.
 
 In the above example the `runLocalConfigFile` setting is shown to point to a local.conf in the taxi-ride-fare example project.
+
+== Azure Blob Storage
+
+Flink supports a number of file systems for reading and writing data and for its state backend. In this section we will look at how to use Microsoft's Azure Blob Storage for these purposes.
+
+Please refer to the Flink documentation here https://ci.apache.org/projects/flink/flink-docs-stable/ops/filesystems/azure.html
+
+To make this work in a Cloudflow context there are a couple of things you need to do as follows.
+
+=== Checkpoints
+
+In the configuration file you need to specify that the `state.backend` is `filesystem`, set `state.checkpoints.dir` to match the the location of your Blob, and provide your access key.
+
+[source,hocon]
+----
+cloudflow.runtimes.flink.config {
+  flink.state.backend = filesystem
+  flink.state.checkpoints.dir = "<your-container>@$<your-azure-account>.blob.core.windows.net/<object-path>"
```

Comment:
Should the URI start with the `wasb://` scheme? 
```
flink.state.checkpoints.dir =  "wasb://<your-container>@$<your-azure-account>.blob.core.windows.net/<object-path>"
```

As shown in the Flink documentation you referred to [Azure Blob Storage](https://ci.apache.org/projects/flink/flink-docs-stable/ops/filesystems/azure.html)

Reply:
Yep thanks